debian/kafka-connect-base/include/etc/confluent/docker/ensure

```diff
@@ -21,7 +21,7 @@ set -o nounset \
 
 echo "===> Check if Kafka is healthy ..."
 
-if [[ -n "${CONNECT_SECURITY_PROTOCOL-}" ]] && [[ $CONNECT_SECURITY_PROTOCOL = "SSL" ]]
+if ([[ -n "${CONNECT_SECURITY_PROTOCOL-}" ]] && [[ $CONNECT_SECURITY_PROTOCOL = "SSL" ]]) || ([[ -n "${CONNECT_SECURITY_PROTOCOL-}" ]] && [[ $CONNECT_SECURITY_PROTOCOL = "SASL_SSL" ]])
```

Comment:
I would prefer to have a ! PlAINTEXT check instead of checking all the secure protocols